crates/router/tests/connectors/wise.rs

```diff
@@ -1 +1,167 @@
+    connector_auth,
+    utils::{self, ConnectorActions, PaymentInfo},
+};
+
+struct WiseTest;
+impl ConnectorActions for WiseTest {}
+impl utils::Connector for WiseTest {
+    fn get_data(&self) -> types::api::ConnectorData {
+        use router::connector::Wise;
+        types::api::ConnectorData {
+            connector: Box::new(&Wise),
+            connector_name: types::Connector::Wise,
+            get_token: types::api::GetToken::Connector,
+        }
+    }
+
+    fn get_auth_token(&self) -> types::ConnectorAuthType {
+        types::ConnectorAuthType::from(
+            connector_auth::ConnectorAuthentication::new()
+                .wise
+                .expect("Missing connector authentication configuration"),
+        )
+    }
+
+    fn get_name(&self) -> String {
+        "wise".to_string()
+    }
+}
+
+impl WiseTest {
+    fn get_payment_info() -> Option<PaymentInfo> {
+        Some(PaymentInfo {
+            address: Some(PaymentAddress {
+                billing: Some(Address {
+                    address: Some(AddressDetails {
+                        country: Some(api_models::enums::CountryAlpha2::GB),
+                        city: Some("London".to_string()),
+                        zip: Some(Secret::new("10025".to_string())),
+                        line1: Some(Secret::new("50 Branson Ave".to_string())),
+                        ..Default::default()
+                    }),
+                    phone: None,
+                }),
+                ..Default::default()
+            }),
+            country: Some(api_models::enums::CountryAlpha2::NL),
+            currency: Some(enums::Currency::GBP),
+            ..Default::default()
+        })
+    }
+}
+
+static CONNECTOR: WiseTest = WiseTest {};
+
+/******************** Payouts test cases ********************/
+// Creates a recipient at connector's end
+#[cfg(feature = "payouts")]
+#[actix_web::test]
+async fn should_create_payout_recipient() {
+    let mut payment_info = WiseTest::get_payment_info();
+    payment_info = payment_info.map(|mut pi| {
+        pi.payout_method_data = Some(api::PayoutMethodData::Bank(api::payouts::BankPayout::Bacs(
+            api::BacsBankTransfer {
+                bank_sort_code: "231470".to_string(),
+                bank_account_number: "28821822".to_string(),
+                bank_name: "Deutsche Bank".to_string(),
+                bank_country_code: enums::CountryAlpha2::NL,
+                bank_city: "Amsterdam".to_string(),
+            },
+        )));
+        pi
+    });
+    let response = CONNECTOR
+        .create_payout_recipient(enums::PayoutType::Bank, payment_info)
+        .await
+        .expect("Payout recipient creation response");
+    assert_eq!(
+        response.status.unwrap(),
+        enums::PayoutStatus::RequiresCreation
+    );
+}
+
+// Create BACS payout
+#[cfg(feature = "payouts")]
+#[actix_web::test]
+async fn should_create_bacs_payout() {
+    let mut payment_info = WiseTest::get_payment_info();
```

Comment:
add all your default data for payout within get_payout_info and use it in all other places below
```suggestion
    let mut payout_info = WiseTest::get_payout_info();
```